src/include/network/routines.rb

```diff
@@ -746,6 +746,17 @@ def has_carrier?(dev_name)
       ).to_i != 0
     end
 
+    def has_physical_port_id?(dev_name)
+      !physical_port_id(dev_name).empty?
+    end
+
+    def physical_port_id(dev_name)
```

Comment:
Yeah you are right, although it is just a line of code and the method name is very illustrative it could be not so clear for someone who doesn't know the behavior.